config/locales/models/assessments/en.yml

```diff
@@ -4,6 +4,12 @@ en:
     attributes:
       assessment:
         max_mark: Out Of
+    errors:
+      models:
+        assessment:
+          attributes:
+            short_identifier:
+              format: short_identifier must only contain alphanumeric, hyphen, or underscore
```

Comment:
Add the word "characters" to the end of the message (again, consistently for all of the messages that discuss the types of allowed characters